composer.json

```diff
@@ -63,7 +63,8 @@
         "nyholm/psr7-server": "1.0.1",
         "lcobucci/jwt": "dev-342php8 as 3.4.9",
         "psr/log": "1.1.3",
-        "monolog/monolog": "2.1.1"
+        "monolog/monolog": "2.1.1",
+        "openemr/wkhtmltopdf-openemr": "^1.0"
```

Comment:
i think this thing is huge (why it is separated). plus, pretty sure it does not work on alpine os.

Reply:
@bradymiller sorry, I was testing the institutional claims... couldn't get it to work on Mac OS either. Maybe we should have platform-based composer files? When you build from composer you can specify composer like "COMPOSER=composer-alpine.json composer install"

Reply:
@kchapple , No prob if just doing this for development/testing purpose. We'll just need to remember to remove it before bringing this into codebase. The goal at some point is to get the institutional billing away from using this pdf package.